packages/cli/src/models/network/TransactionController.ts

```diff
@@ -55,18 +56,27 @@ export default class TransactionController {
     }
   }
 
-  public async sendTransaction(proxyAddress: string, methodName: string, methodArgs: string[]): Promise<any> {
-    if (!this.networkFile.hasProxies({ address: proxyAddress })) {
-      throw Error(`Proxy at address ${proxyAddress} not found.`);
-    }
+  public async callContractMethod(proxyAddress: string, methodName: string, methodArgs: string[]): Promise<void | never> {
+    const { method, contract } = this.getContractAndMethod(proxyAddress, methodName, methodArgs);
+    try {
+      log.info(`Calling: ${callDescription(method, methodArgs)}`);
+      let result = await contract.methods[methodName](...methodArgs).call({ ...this.txParams });
+      if (typeof result === 'object' && Object.keys(result).length !== 0) {
+        result = `(${Object.values(result).join(', ')})`;
+      }
 
-    const { package: packageName, contract: contractName } = this.networkFile.getProxy(proxyAddress);
-    const contractManager = new ContractManager(this.packageFile);
-    const contract = contractManager.getContractClass(packageName, contractName).at(proxyAddress);
-    const { method } = buildCallData(contract, methodName, methodArgs);
+      isUndefined(result) || typeof result === 'object'
```

Comment:
Be careful with the typeof object check: if the result is an array, typeof an array is object, so the result will not be shown. Same applies to bignumbers (web3.js returns strings in the version 37, but have rolled back to bns in a more recent one). Also, I'm not sure what happens when returning structs.